packages/devextreme/testing/tests/DevExpress.ui.widgets/mapParts/azureTests.js

```diff
@@ -466,18 +466,25 @@ QUnit.module('basic options', moduleConfig, () => {
     QUnit.test('Should add onClick handler with correct args', function(assert) {
         const done = assert.async();
         let clickFired = 0;
+        const originalEvent = new PointerEvent({ type: 'click' });
 
         const map = $('#map').dxMap({
             provider: 'azure',
             onClick: (e) => {
                 assert.strictEqual(e.component, map, 'click event includes component instance');
                 assert.strictEqual($(e.element).is($('#map')), true, 'click event includes root element');
                 assert.deepEqual(e.location, { lat: 88, lng: 88 }, 'click event includes correct location');
+                assert.ok(e.event instanceof PointerEvent, 'click event includes event instance');
+                assert.deepEqual(e.event, originalEvent, 'click event is equal to passed originalEvent');
```

Comment:
Our goal is to check that it's the same even which was displatched by original map, not just an event with the same data
```suggestion
                assert.strictEqual(e.event, originalEvent, 'click event is equal to passed originalEvent');
```

Reply:
Fixed